packages/devtools_app/lib/src/screens/logging/_message_column.dart

```diff
@@ -24,20 +22,20 @@ class MessageColumn extends ColumnData<LogData>
   bool get supportsSorting => false;
 
   @override
-  String getValue(LogData dataObject) =>
+  String? getValue(LogData dataObject) =>
```

Comment:
make this return a non-nullable string. We may need to change the return type of `ColumnData.getValue` to `Object` so that it is non-nullable by default. The table classes and other column classes use `getDisplayValue` without null checking, and `getDisplayValue` uses `getValue` without null checking, assuming the value returned by `getValue` is non-null. So if we return a nullable value from `getValue`, this will likely cause some exceptions to be thrown from tables.